### ISSUE TITLE
make UI responsive when reducing window width during extraction phase

### ISSUE THREAD

**Issue by nicoalee** [neurostuff/neurostore#556]
Currently, when shrinking the width of the UI during the extraction phase, the UI does not respond well. This is important as a lot of people look at the full text while editing.

**Comment by adelavega** [comment]
@nicoalee still an issue?

**Comment by nicoalee** [comment]
yeah it is. The handsontable spreadsheet and the actions panel are compressed when the window shrinks.

<img width="865" height="987" alt="Image" src="https://github.com/user-attachments/assets/4dcaa65e-c200-467f-9e79-e3b11528aabb" />
